contentmgr/shuttle.go

```diff
@@ -220,6 +220,13 @@ func (cm *ContentManager) processShuttleMessage(handle string, msg *drpc.Message
 			cm.log.Errorf("handling split complete message from shuttle %s: %s", handle, err)
 		}
 		return nil
+	case drpc.OP_SanityCheck:
+		sc := msg.Params.SanityCheck
+		if sc == nil {
+			return ErrNilParams
+		}
+		cm.HandleSanityCheck(sc.CID, sc.Err)
```

Comment:
I think this is going to be an expensive operation (it looks up obj and obj ref tables). I’m thinking if we should decouple this as part of the GC.

Reply:
I do not think so, we do this all the time when we do commp, plus missing blocks do not happen often (and should not)